pkg/resources/base/common.go

```diff
@@ -27,11 +27,16 @@ import (
 )
 
 const (
-	componentName      = "common"
-	istioReaderName    = "istio-reader"
-	IstioConfigMapName = "istio"
+	componentName                 = "common"
+	istioReaderName               = "istio-reader"
+	istioReaderServiceAccountName = istioReaderName + "service-account"
```

Comment:
`istioReaderServiceAccountName = istio-readerservice-account`
Shouldn't it have a `-` in between?

Reply:
True dat, nice catch, fixed, thank you! :)